src/sentry/auth/helper.py

```diff
@@ -727,7 +727,7 @@ def __init__(
         self.organization: RpcOrganization = self.organization
         self.provider: Provider = self.provider
 
-    def get_provider(self, provider_key: str | None) -> PipelineProvider:
+    def get_provider(self, provider_key: str | None, **kwargs) -> PipelineProvider:
```

Comment:
fix tests since we are passing org to `get_provider`. will remove after we fix the feature flag